apps/server/src/modules/account/domain/services/account.service.ts

```diff
@@ -348,6 +348,10 @@ export class AccountService extends AbstractAccountService implements DeletionSe
 		return new Account({ ...ret.getProps(), idmReferenceId: idmAccount?.idmReferenceId });
 	}
 
+	async updateLastLogin(accountId: string, lastLogin: Date): Promise<void> {
```

Comment:
Wollen wir lastLogin von außen steuern, oder eher updateLastLogin(accountId: string)? So das Date.now() dann in der Methode ausgeführt wird.